apis/fluent-en_GB/atrium-api-fluent-en_GB/src/jvmMain/kotlin/ch/tutteli/atrium/api/fluent/en_GB/chronoZonedDateTimeExpectations.kt

```diff
@@ -41,8 +39,7 @@ fun <T : ChronoZonedDateTime<T>> Expect<T>.toBeLessThanOrEqualTo(expected: T): N
 )
 fun <T : ChronoZonedDateTime<T>> Expect<T>.notToBeGreaterThan(expected: T): Nothing =
     throw PleaseUseReplacementException(
-        "" +
-            "If you wish to use deprecated methods for ChronoZonedDateTime functions" +
+        "If you wish to use deprecated methods for ChronoZonedDateTime functions" +
             ", use a feature extractor "
```

Comment:
nitpick: have not seen it before, could you move the `,` on the previous line please (also in other places)

Reply:
Fixed by moving the whole message to a single line if that's okay?

Reply:
That's good, thanks